WordPress/Classes/Extensions/UIImageView+Gravatar.swift

```diff
@@ -155,4 +209,96 @@ extension UIImageView {
         static let imageSize = 80
         static let rating = GravatarRatings.g
     }
+
+    /// Returs the proper download URL for the provided icon
+    ///
+    /// Parameters:
+    ///  - siteIcon: the icon's path
+    fileprivate func URLWithSiteIcon(_ siteIcon: String) -> URL? {
+        if self.isPhotonURL(siteIcon) || self.isWordPressComFilesURL(siteIcon) {
+            return self.siteIconURLForSiteIconUrl(siteIcon)
+        } else if self.isBlavatarURL(siteIcon) {
```

Comment:
No need for `else`, since the block above returns. (Same comment below!). + No need for `self.`